maxim/logger/mistral/client.py

```
@@ -0,0 +1,215 @@
+
+from mistralai.sdk import Mistral
+from mistralai.chat import Chat
+from mistralai.models import CompletionEvent
+
+from ...scribe import scribe
+from ..logger import Generation, GenerationConfig, Logger, Trace, TraceConfig
+from .utils import MistralUtils
+
+
+class MaximMistralChat:
+    def __init__(self, chat: Chat, logger: Logger):
```

Ruff (0.11.9)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)